Frontend-Angular/data-polish-frontend/src/assets/Dataprofile_config.json

```diff
@@ -11,29 +11,89 @@
         
         
     },
-    "NUMBEROFFIELDS":{
-        "WHAT":"Displays the count of fields in the dataset.",
-        "WHY":""
+    "NUMBEROFRECORDS":{
+        "WHAT":"Displays the count of Records in the dataset.",
+        "WHY":"In order to understand the Dataset size"
         
         
     },
-    "NUMBEROFFIELDS":{
-        "WHAT":"Displays the count of fields in the dataset.",
-        "WHY":""
+    "bubble-chart":{
+        "WHAT":"Visualises the categorical features within a column as bubbles.",
+        "WHY":"Use to identify categorical distribution. Large bubbles may indicate asimetric data points needing normalisation."
         
         
     },
-    "NUMBEROFFIELDS":{
-        "WHAT":"Displays the count of fields in the dataset.",
-        "WHY":""
+    "cardinality":{
+        "WHAT":"Shows the uniqueness of data by representing the number of distinct values in a column.",
```

Comment:
High cardinality can introduce the curse of dimensionailty, columns may need or encoded if using them as a predictor variable in ML